config/README.rst

```diff
@@ -142,21 +142,41 @@ updated. Example:
 .. code-block:: ini
 
     [meta]
-    template = pure-python
-    commit-id = < commit-hash >
+    template = "pure-python"
+    commit-id = "< commit-hash >"
+
+    [python]
+    with-legacy-python = true
+    with-pypy = false
+    with-docs = true
+    with-sphinx-doctests = false
+
+    [coverage]
     fail-under = 98
-    with-pypy = False
-    with-docs = True
-    with-sphinx-doctests = False
-    with-legacy-python = True
-    additional-manifest-rules =
-    additional-flake8-config =
-      ignore = D203
-    additional-check-manifest-ignores =
 
+    [flake8]
+    additional-config = [
+        "# E221 multiple spaces before operator",
+        "# E222 multiple spaces after operator",
+        "per-file-ignores =",
+        "    src/foo/bar.py: E221 E222"
+        "ignore = D203",
+    ]
+
+    [manifest]
+    additional-rules = [
+        "include *.foo",
+        "include *.bar",
+        ]
+
+    [check-manifest]
+    additional-ignores = [
+        "docs/html/*",
+        "docs/source/_static/*",
+        ]
```

Comment:
The different alignment of the `]` in the three examples offends my inner pedant.

Reply:
I have now aligned them in the way the file would be written by `config-package.py`.